README.md

```diff
@@ -158,15 +158,16 @@ You would not like to push a freeze period of 2 seconds in the repository!
 
 ### Deployment
 
-The latest version of this code lives on the `eosforumrcpp` account on
-the EOS Mainnet and on the `cancancan345` account on the EOS Kylin network.
+The latest version of this code lives on the `eosio.forum` account on
+the EOS Mainnet.
 
-Tools that initially integrated support for this contract are:
-- [eosc](https://github.com/eoscanada/eosc) has a command-line interface
-  implementation to submit posts and votes.
-- [EOS ToolKit Forum Post](https://www.myeoskit.com/tools/referendums) allows you to post content through this contract.
-- MyEOSKit already has special casing for the `post` actions. See
-  [this transaction for example](https://www.myeoskit.com/?#/tx/c40e30d70ee92a0f57af475a828917851aa62b01bfbf395efae5c1a2b22068f0).
+There is also a few accounts that were used for development purposes
```

Comment:
@joshkauffman I added this section, wondering if it's really usefull. I though it could be intersting as an historical thing.

Thoughts?